diff -urN plib-1.8.4/src/ssg/Makefile.am plib-1.8.4.new/src/ssg/Makefile.am
--- plib-1.8.4/src/ssg/Makefile.am	2004-10-30 00:46:25.000000000 +0200
+++ plib-1.8.4.new/src/ssg/Makefile.am	2006-11-05 15:41:49.000000000 +0100
@@ -29,6 +29,8 @@
 	ssgSaveIV.cxx ssgAnimTransform.cxx\
 	ssgVertSplitter.h ssgVertSplitter.cxx ssgStatistics.cxx
 
+libplibssg_la_LIBADD = ../sg/libplibsg.la ../util/libplibul.la
+
 INCLUDES = -I$(top_srcdir)/src/sg -I$(top_srcdir)/src/util
 
 endif
--- plib-1.8.4/src/net/Makefile.am~	2007-05-27 22:30:14.521956341 +0200
+++ plib-1.8.4/src/net/Makefile.am	2007-05-27 22:32:20.483299305 +0200
@@ -8,6 +8,8 @@
 libplibnet_la_SOURCES = netBuffer.cxx netChannel.cxx netChat.cxx \
 	netMessage.cxx netMonitor.cxx netSocket.cxx
 
+libplibnet_la_LIBADD = ../util/libplibul.la
+
 INCLUDES = -I$(top_srcdir)/src/util
 
 endif
