diff -urN plib-1.3.1.org/configure.in plib-1.3.1/configure.in
--- plib-1.3.1.org/configure.in	Fri Apr 20 17:42:45 2001
+++ plib-1.3.1/configure.in	Fri Apr 20 17:46:55 2001
@@ -146,12 +146,12 @@
     CFLAGS="$CFLAGS -I/usr/local/include -L/usr/local/lib"
     CXXFLAGS="$CXXFLAGS -I/usr/local/include -L/usr/local/lib"
   else
-    CFLAGS="$CFLAGS -O6 -Wall"
-    CXXFLAGS="$CXXFLAGS -O6 -Wall"
+    CFLAGS="$CFLAGS -Wall"
+    CXXFLAGS="$CXXFLAGS -Wall"
   fi
 else
-  CFLAGS="$CFLAGS -O6 -Wall $MACH -malign-double -I/usr/local/include -L/usr/local/lib"
-  CXXFLAGS="$CXXFLAGS -O6 -Wall $MACH -malign-double -I/usr/local/include -L/usr/local/lib"
+  CFLAGS="$CFLAGS -Wall $MACH -malign-double -I/usr/local/include -L/usr/local/lib"
+  CXXFLAGS="$CXXFLAGS -Wall $MACH -malign-double -I/usr/local/include -L/usr/local/lib"
 fi
 
 dnl Checks for typedefs, structures, and compiler characteristics.
@@ -165,5 +165,6 @@
 	src/sg/Makefile \
 	src/sl/Makefile \
 	src/ssg/Makefile \
+	src/ssgAux/Makefile \
 	src/fnt/Makefile )
 
